sdk/examples/wallet/08_send_nft.rs

```diff
@@ -29,7 +32,7 @@ async fn main() -> Result<()> {
             .await?;
 
         let outputs = vec![SendNftParams {
-            address: "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu".to_string(),
+            address: Bech32Address::try_from_str("rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu")?,
```

Comment:
Just thinking out loud: is there no way to make this a bit better UX wise? What if we have a `SendNftParams` constructor that takes a Bech32AddressLike? It would make it fallible but we already have a fallible thing here, I don't think it would matter much but would make the overall thing a bit better?

Reply:
Yes, we could do that. Do you think it's worth adding even more changes to this PR?

Reply:
You can open a PR to do it later that's also fine, but I think we should do this